### PR TITLE
Restructure the package

### PR DESCRIPTION
**Pull request by liyanhui1228**
For #54

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -14,27 +14,29 @@
...
         'Intended Audience :: Developers',
         'License :: OSI Approved :: Apache Software License',
         'Programming Language :: Python',
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: Implementation :: CPython',
+        'Programming Language :: Python :: 3.6',
```
Comment (duggelz) [inline setup.py]: I think you accidentally replaced Python 3.5 here, instead of adding 3.6 on the line below.

File: nox.py [hunk 2]
```diff
@@ -12,9 +12,100 @@
...
 
-import os
+from __future__ import absolute_import
 
 import nox
+import os
+
+
+@nox.session
+@nox.parametrize('python_version', ['2.7', '3.4', '3.5', '3.6'])
```
Comment (duggelz) [inline nox.py]: The google-cloud-python packages did 'python_version' -> 'py', 'system_tests' -> 'system', 'unit_tests' -> 'unit'.  Do you have any opinion on doing the same here?  I think I'm fine with either.

File: nox.py [hunk 3]
```diff
@@ -12,9 +12,100 @@
...
+
+    # Install all test dependencies, then install this package into the
+    # virutalenv's dist-packages.
+    session.install('-r', 'requirements-test.txt')
+    session.install('.')
+
+    # Run py.test against the system tests.
+    session.run(
+        'py.test',
+        '-vvv',
```
Comment (duggelz) [inline nox.py]: Is this deliberately verbose, or a debugging line left in by accident?

File: setup.py (hunk 1 above)
Comment (liyanhui1228) [reply setup.py]: Good catch, added back.

File: nox.py (hunk 3 above)
Comment (liyanhui1228) [reply nox.py]: It's verbose to better see what happens in the system tests, but we can remove it for now.

File: nox.py (hunk 2 above)
Comment (liyanhui1228) [reply nox.py]: Changed.

**Pull request merged**